### ISSUE TITLE
Operations inside return statements are not being detected.

### ISSUE THREAD

**Issue by Ryan1729** [tommy-muehle/go-mnd#11]
Currently no magic numbers are detected in the following code:

```golang
func foo(x int32) int32 {
	return x + 42
}
```

Even though the magic number is detected in this code:

```golang
func foo() int32 {
	return 42
}
```

**Comment by tommy-muehle** [comment]
@Ryan1729 
Thanks for the report.

**Comment by tommy-muehle** [comment]
@Ryan1729 Thanks again for your report. I fixed this bug with version 2.0.2. Feel free to try it out :)
